ci/ray_ci/automation/wheels_pypi.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+if [[ -z "$RAY_HASH" ]]; then
+    echo "RAY_HASH env var should be provided"
+    exit 1
+fi
+
+if [[ -z "$RAY_VERSION" ]]; then
+    echo "RAY_VERSION env var should be provided"
+    exit 1
+fi
+
+TEST_PYPI_TOKEN="test_pypi_token"
+
+download_wheel() {
+    # Download single wheel from given URL into TMP_DIR/<wheel_filename>.whl
+    WHEEL_URL=$1
+    OUTPUT_FILE="${TMP_DIR}/${WHEEL_URL##*/}"
+    if [ "${OVERWRITE-}" == "1" ] || [ ! -f "${OUTPUT_FILE}" ]; then
+        wget -O "${OUTPUT_FILE}" "${WHEEL_URL}"
+    fi
+}
+
+download_wheels() {
+    # Iterate through all Python versions, platforms, and ray types and download wheels
+    python_versions=("cp38-cp38" "cp39-cp39" "cp310-cp310" "cp311-cp311")
+    platforms=("manylinux2014_x86_64" "manylinux2014_aarch64" "macosx_10_15_x86_64" "macosx_11_0_arm64" "win_amd64")
+    ray_types=("ray" "ray_cpp")
+
+    url_prefix="https://ray-wheels.s3-us-west-2.amazonaws.com/releases"
+    for platform in "${platforms[@]}"; do
+        for ray_type in "${ray_types[@]}"; do
+            for python_version in "${python_versions[@]}"; do
```

Comment:
I feel that we should have a centralized python script to generate this kind of arrays. this is re-defined in so many different places.